Adyen/Components/Base/PresentableComponent.swift

```diff
@@ -13,8 +13,17 @@ public protocol Localizable {
     var localizationParameters: LocalizationParameters? { get set }
 }
 
+/// :nodoc:
+/// Represents any object than can handle a cancel event.
+public protocol Cancellable {
+    
+    /// :nodoc:
+    /// Called when the user cancels the component.
+    func didCancel()
+}
+
 /// A component that provides a view controller for the shopper to fill payment details.
-public protocol PresentableComponent: Component {
+public protocol PresentableComponent: Component, Cancellable {
```

Comment:
Could you explain?
I like an idea, but I don't really see that it is used anywhere...

Reply:
`PollingAwaitComponent` use it to stop polling.